sdk/python/feast/infra/offline_stores/offline_store.py

```diff
@@ -163,6 +163,23 @@ def metadata(self) -> Optional[RetrievalMetadata]:
         """
         pass
 
+    def supports_remote_storage_export(self) -> bool:
+        """
+        This method should return True if the RetrievalJob supports `to_remote_storage()`.
+        """
+        return False
+
+    def to_remote_storage(self) -> List[str]:
```

Comment:
nit: can you indicate what the `List[str]` (presumably paths?) in the docstring

Reply:
yup